doctor_visits/delphi_doctor_visits/geo_maps.py

```diff
@@ -160,37 +165,11 @@ def county_to_megacounty(self, data, threshold_visits, threshold_len):
             recent_visits_df, how="left", on=["ServiceDate", "PatCountyFIPS"]
         )
 
-        # mark date-fips points to exclude if we see less than threshold visits that day
-        data["ToExclude"] = data["RecentVisits"] < threshold_visits
-
-        # now to convert to megacounties
-        state_map = pd.read_csv(
-            join(self.geo_filepath, "02_20_uszips.csv"),
-            usecols=["fips", "state_id"],
-            dtype={"state_id": str},
-            converters={"fips": GeoMaps.convert_fips},
-        )
-        state_map.drop_duplicates(inplace=True)
-        data = data.merge(
-            state_map, how="left", left_on="PatCountyFIPS", right_on="fips"
-        )
-        # drops rows with no matches, which should not be many
-        data.dropna(inplace=True)
-        data.drop(columns=["fips"], inplace=True)
-        data["StateFIPS"] = data["PatCountyFIPS"].apply(
-            lambda f: str((int(f) // 1000) * 1000).zfill(5)
-        )
-
-        megacounty_df = (
-            data[data["ToExclude"]]
-            .groupby(["ServiceDate", "StateFIPS"])
-            .sum()
-            .reset_index()
-        )
-        megacounty_df["ToExclude"] = False
```

Comment:
I think the values where ToExclue=True were meant to be filtered out but weren't here.

Reply:
I'm not sure. In line 185, we subset by the ["ToExclude"] column, which should have removed all the cases we wanted to filter out. The line at 190 may just be redundant?